docs/csharp/toc.yml

```diff
@@ -1,14 +1,16 @@
 - name: C# documentation
   href: index.yml
-- name: Get Started
+- name: Get started
   items:
   - name: Overview
+    displayName: get started
     href: getting-started/index.md
   - name: Introduction to the C# language and the .NET framework
     href: getting-started/introduction-to-the-csharp-language-and-the-net-framework.md
 - name: Tutorials
   items:
   - name: Overview
+    displayName: tutorials
```

Comment:
```suggestion
    displayName: Tutorials
```

Reply:
No, the capitalization doesn't matter. It's not actually displayed anywhere (despite the name).

Reply:
@gewarren, The name gave me the impression that it's displayed somewhere. I'm curious what this key does ?

Reply:
It's basically to add keywords to the node. The TOC filter shows that node if the user filters on that keyword.